### PR TITLE
Change new module function naming to be consistent across modules

### PR DESCRIPTION
**Pull request by madhuravi**

### CODE REVIEW

File: service/doc.go [hunk 1]
```diff
@@ -79,7 +79,7 @@
 //       kafka.Module("kakfa_topic1", []string{"worker"}),
 //     ).WithModule(
 //       "keyvalue",
-//       rpc.ThriftModule(rpc.CreateThriftServiceFunc(NewYarpcThriftHandler)),
```
Comment (anuptalwalkar) [inline service/doc.go]: Didnt catch it before, but name should be "yarpc" not "keyvalue"

File: service/README.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ func main() {
     kafka.Module("kakfa_topic1", []string{"worker"}),
   ).WithModule(
     "keyvalue",
```
Comment (anuptalwalkar) [inline service/README.md]: name: yarpc

File: modules/task/README.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ import (
 func main() {
   svc, err := service.WithModule(
     "example",
```
Comment (anuptalwalkar) [inline modules/task/README.md]: name: task?

**Comment by madhuravi** [comment]
Wow, all green looks strange and new ;)

**Pull request merged**